### PR TITLE
Broadcaster parameters

### PR DESCRIPTION
**Pull request by christophfroehlich**
`imu_sensor_broadcaster` and `force_torque_sensor_broadcaster` had parameters manually parsed for being non-empty. I added them to the generate_parameter_library yaml and updated the tests accordingly.

### CODE REVIEW

**Review by destogl — APPROVED** [review]
Thanks!

File: force_torque_sensor_broadcaster/test/test_force_torque_sensor_broadcaster.cpp [hunk 1]
```diff
@@ -315,3 +292,12 @@ TEST_F(ForceTorqueSensorBroadcasterTest, All_InterfaceNames_Publish_Success)
   ASSERT_EQ(wrench_msg.wrench.torque.y, sensor_values_[4]);
   ASSERT_EQ(wrench_msg.wrench.torque.z, sensor_values_[5]);
 }
+
+int main(int argc, char ** argv)
+{
+  ::testing::InitGoogleTest(&argc, argv);
```
Comment (bmagyar) [inline force_torque_sensor_broadcaster/test/test_force_torque_sensor_broadcaster.cpp:298]: we use Google Mock in all:

```suggestion
  ::testing::InitGoogleMock(&argc, argv);
```

**Review by bmagyar — CHANGES_REQUESTED** [review]

File: force_torque_sensor_broadcaster/test/test_load_force_torque_sensor_broadcaster.cpp [hunk 2]
```diff
@@ -43,6 +41,13 @@ TEST(TestLoadForceTorqueSensorBroadcaster, load_controller)
       "test_force_torque_sensor_broadcaster",
       "force_torque_sensor_broadcaster/ForceTorqueSensorBroadcaster"),
     nullptr);
+}
 
+int main(int argc, char ** argv)
+{
+  ::testing::InitGoogleTest(&argc, argv);
```
Comment (christophfroehlich) [inline force_torque_sensor_broadcaster/test/test_load_force_torque_sensor_broadcaster.cpp:48]:
```suggestion
  ::testing::InitGoogleMock(&argc, argv);
```

File: imu_sensor_broadcaster/test/test_imu_sensor_broadcaster.cpp [hunk 3]
```diff
@@ -208,3 +178,12 @@ TEST_F(IMUSensorBroadcasterTest, SensorName_Publish_Success)
     EXPECT_EQ(imu_msg.linear_acceleration_covariance[i], 0.0);
   }
 }
+
+int main(int argc, char ** argv)
+{
+  ::testing::InitGoogleTest(&argc, argv);
```
Comment (christophfroehlich) [inline imu_sensor_broadcaster/test/test_imu_sensor_broadcaster.cpp:184]:
```suggestion
  ::testing::InitGoogleMock(&argc, argv);
```

File: imu_sensor_broadcaster/test/test_load_imu_sensor_broadcaster.cpp [hunk 4]
```diff
@@ -42,6 +40,13 @@ TEST(TestLoadIMUSensorBroadcaster, load_controller)
     cm.load_controller(
       "test_imu_sensor_broadcaster", "imu_sensor_broadcaster/IMUSensorBroadcaster"),
     nullptr);
+}
 
+int main(int argc, char ** argv)
+{
+  ::testing::InitGoogleTest(&argc, argv);
```
Comment (christophfroehlich) [inline imu_sensor_broadcaster/test/test_load_imu_sensor_broadcaster.cpp:47]:
```suggestion
  ::testing::InitGoogleMock(&argc, argv);
```

**Review by bmagyar — APPROVED** [review]
Thank you!

**Pull request merged**